Header header
int16[16] raWchannels
int16[16] mappedChannels
bool failsafe
bool frame_lost
